articlewithCommentCount.sql

```diff
@@ -0,0 +1,10 @@
+\c nc_news_test
+
+SELECT articles.*, COUNT(comments.article_id)::INTEGER AS
```

Comment:
What's going on in here then? If it was just so you can look at what the query was giving back then good idea! But it should be removed now you're done with it please.

Reply:
yeah it was just to see what was going on! I shall deleted :D